### PR TITLE
i104 Add Alerts

### LINKED ISSUE #104: Add Alerts
## Basic Information
Successful and unsuccessful alerts for all create, update and delete requests.
Alerts need to be useable
https://vuetifyjs.com/en/components/alerts/

### PR DESCRIPTION
**Pull request by wuryantoAji**
## Change Summary
Adding alert component 

### Change Form

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
It seems I can't login to the web using the django superuser so I created two recording both is shown when login failed but one gives error alert while the other one gives success alert
https://drive.google.com/file/d/1xINVS2UfsVpXjlrvOhjGDtI6QdD5SO31/view?usp=sharing -> error alert
https://drive.google.com/file/d/1_5DTzAuyHw7hzBpCfbTEt2D7ebfcMz2S/view?usp=sharing -> success alert

##Update
now use this library for the notification
https://github.com/kyvg/vue3-notification

haven't check on the success notification, I might need an account to check whether the success notification is correct or not. 

> another thing is that I haven't add any notification on EventsModal or any components with function like these, kindly give me pointer on to add notification for these kind of functions. thanks in advance
> 
> ```
> const addEvent = () => eventStore.createEvent(refs()).then(closeModal).catch(console.log)
> ```
got a maybe working solution for this, will work on it. probably done on Wednesday before stand up

# Related Issue

- Resolve #104

### CODE REVIEW

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
Please add alert to all backend requests as discussed, thank you :)

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
Please do for all requests. Event, Team, Mileage, User settings page.

Merge main into your branch first because a few changes have been added.

**Comment by wuryantoAji** [comment]
Hello, I think I have put all notify on the function that calls backend. Please kindly tell me if there is something that I miss or if a notification is too annoying/not necessary for the function. thanks in advance

File: client/src/stores/team.ts [hunk 1]
```diff
@@ -21,40 +22,91 @@ export const useTeamStore = defineStore('team', {
     async getTeams() {
       const teams = await server.get('team/get_teams/').then((res) => {
         if (res.status == 200) {
+          notify({
+            title: 'Get Teams',
+            type: 'success',
+            text: 'Get Teams Successful'
+          })
```
Comment (ChantelleKerr) [inline client/src/stores/team.ts:29]: We probably dont need this alert. Keep the unsuccessful one though

File: client/src/stores/team.ts [hunk 2]
```diff
@@ -21,40 +22,91 @@ export const useTeamStore = defineStore('team', {
...
+      await server
+        .get(`team/get/${teamId}/`)
+        .then((res) => {
+          if (res.status == 200) {
+            const data = camelize(res.data) as Object as Team
+            this.currentTeam = JSON.stringify(data)
+            notify({
+              title: 'Get Team',
+              type: 'success',
+              text: 'Get Team Successful'
```
Comment (ChantelleKerr) [inline client/src/stores/team.ts:51]: We dont need this - keep unsuccessful one though

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
We can remove two of the successful one because we dont want it to pop up all the time. Also, the position of the notification on mobile doesnt look right. You could probably make the notification have a smaller width.

Good job its working as expected :)

**Comment by ChantelleKerr** [comment]
Is this ready for review again? :)

**Comment by wuryantoAji** [comment]
not yet chantelle, I will re-request for review if it's ready

**Review by ChantelleKerr — APPROVED** [review]
looks good, nice work

**Pull request merged**